Website (#80)
popstarfreas/DG-TS: docs/FAQ/ServernGamemodes.md

---
title: Server
parent: FAQ
nav-order: 4
---
# Server

The server has a few Dimensions. You can join other dimensions by doing `/<dimension name>`, if you want to join Survival, you'd do `/survival` and if you want to join Build I'd do `/build` and so on. You can also join other dimensions by using the warp plates in Rift.

Minigames : [Zombies](#Zombies)

Fun : [PvE](#PvE), [PvP](#PvP), [Build](#Build), [Survival](#Survival)

Others : [Items](#Items), [Special Items](#Special-Items) and [Archives](#Archives)

Closed : [AutoSurvival](#AutoSurvival)

## Minigames

### Zombies
TShock Server, `/gamemodes` then enter the little pond below Z-O-M-B-I-E-S letter statues to join.

Zombies is the only "minigame" in the server, it involves a wave of monsters every round that you have to survive.

There are 4 types of lobbies:

> Public Lobbies, are normal lobbies created by the server that anyone can join as long as it is in the intermission period up to the 3rd round.

Reviewer: ```suggestion
Public Lobbies, are normal lobbies created by the server that anyone can join as long as it is currently in intermission or below Round 4.
```

<a name="Hardcore-Zombies"></a>

> Hardcore Zombies, hardcore zombies gives more XP and DP but, if you get hit you die instantly, HP Chests are replaced with Life Chests(buy life so you won't die instantly when you get hit), you can join hardcore zombies by doing `/j z -hc` in the gamemodes lobby.

Reviewer: ```suggestion
Hardcore Zombies, hardcore zombies gives more XP and DP but, if you get hit you die instantly, HP Chests are replaced with Life Chests (buy life so you won't die instantly when you get hit), you can join hardcore zombies by doing `/j z -hc` in the gamemodes lobby.
```

> Protected Lobbies(Formerly Private Lobbies), basically a private version of the public lobby, only invited people can join. You can create a protected lobby by doing `/j z -p` in the gamemodes lobby.

Reviewer: ```suggestion
Protected Lobbies (formerly Private Lobbies), basically a private version of the public lobby, only invited people can join. You can create a protected lobby by doing `/j z -p` in the gamemodes lobby.
```

> Private Lobbies(Formerly Custom Lobbies), a lobby you can customize, changing the arena, amount of HP, etc., they are automatically private. You can't use this type of lobby for gaining DP or XP. You can create a private lobby by doing `/j z -c` in the gamemodes lobby to join.

Reviewer: ```suggestion
Private Lobbies (formerly Custom Lobbies), a lobby you can customize, changing the arena, amount of HP, etc., they are automatically private. You can't use this type of lobby for gaining DP or XP. You can create a private lobby by doing `/j z -c` in the gamemodes lobby to join.
```

> Q: How can I join my friend?
>

> > A: You can't.

Reviewer: ```suggestion
> A: They must invite you first.
```

> Q: Can anyone else see the lobby chat? 
>
> A: No, only your lobbymates can see the lobby chat, anyone else in other dimensions can't.

> Q: Is there a way I can speak outside of lobby chat?
>
> A: Yes, simply do `/lobby chat` to toggle between global chat and lobby chat.

> Q: Is there another way to buy huge amounts HP or Mana instead of spamming the chest?
>
> A: Yes, `/lobby upall` to toggle on|off, this will consume your points till the chest is maxed out or you ran out of points.

> Q: How can I give other people my points?
>
> A: Assuming both you and the player you want to give points to are alive, you can do `/lobby ts <Player> <Points>`.
>
> Say I want to give Blanc 1 point, I can do `/lobby ts Bla 1` or `/lobby ts Blanc 1`

> Q: What are abilities?
>
> A: Abilities are like skills.

> Q: Where can I see a list of abilities/information about abilities?
>
> A: [here](https://github.com/popstarfreas/DG-TS/wiki/Zombies-Abilities).

## Fun

### PvE
TShock Server, `/pve` to join.

> PvE is a dimension where you kill mobs and bosses, either for fun or to get DP and XP, it has a few warps in spawn, but if you're do lazy to walk, just do `/warp list` then do `/warp <warp name>`

Reviewer: ```suggestion
PvE is a dimension where you kill mobs and bosses, either for fun or to get DP and XP, it has a few warps in spawn, but if you're to lazy to walk, just do `/warp list` then do `/warp <warp name>`
```

> Q: How can I get unfrozen/unstoned/unwebbed?
>
> A: Login to an account, if you don't already have one, follow the steps in [Registering](Accounts.md#Registering).

> Q: What does "The ancient spirits of light and dark are dying." mean?
>
> A: It means hardmode is about go away, this is purely a server feature, it is not a feature of Terraria.

> Q: What does "The ancient spirits of light and dark have been vanquished." mean?
>
> A: It means hardmode is gone and boss data is wiped, this is purely a server feature, it is not a feature of Terraria.

> Q: How does hardmode die?
>
> A: When the mob cap is reached(if you have [Crew](#Crew) you can do `/worlddata mode` to see the cap and the current kill count).

> Q: What are the warps?
>
> A: (As of Oct 20 2019) The warps are Elevator, cavern, lhell, rhell, hell, locean, rocean, dungeon, jungle. You can see a list of warps by doing `/warp list`. locean and rocean are both ocean warps, locean is for the left ocean and rocean is for the right ocean. lhell, hell, rhell are all hell warps, lhell is for left hell, hell is a point in hell close to the center of the world, rhell is for right hell.

> Q: Where can I build a house?
>

> > A: You can build a house on the spawn platform(the entirety of the asphalt runway), if you build it anywhere else, it will get removed by world regen.

Reviewer: ```suggestion
> A: You can build a house on the spawn platform (the entirety of the asphalt runway), if you build it anywhere else, it will get removed by world regen.
```

> Q: How can I protect it?
>

> > A: You can protect it by using the `/house define` command([Crew](#Crew) is required), after executing the command, it will tell you to hit 2 points in your house, top-left and bottom-right.

Reviewer: ```suggestion
> A: You can protect it by using the `/house define` command ([Crew](#Crew) is required), after executing the command, it will tell you to hit 2 points in your house, top-left and bottom-right.
```

> Q: How many houses can I define?
>
> A: As of Dec 27 2019, 2 houses can be defined by the player.

> Q: How big can my house be?
>
> A: As of Dec 27 2019, houses can occupy an area of 2024 blocks each.

## Others

### Items
TSL Server, `/items` to join.

Free items dimension.

> Q: I can't find the item I want, how can I spawn it in?
>
> A: You can spawn items using the command `/i <item name or id> <stack> <prefix>`.
	
### Special Items
TSL Server, `/specialitems` to join.

> SpecialItems is a dimension where you can modify items, you do not keep the items when you leave the server, the modified items will disappear.

Reviewer: ```suggestion
Special Items is a dimension where you can modify items, you do not keep the items when you leave the server, the modified items will disappear.
```

> Q: Why can't I see my friends?
>
> A: TSL groups players into groups of 10.

> Q: How can I see other people's projectiles?
>
> A: Enable PvP. Caution: they may kill you repeatedly, if this happens, you can do `/rift` to leave to disable pvp and join again.

### Rift
TSL Server, `/rift` to join.

Rift or formerly Mirror is the dimension you are in when you join the server on port 7777, it is the main hub to all these dimensions. You can't see other players in Rift to reduce lag.

### Gamemodes

TSL Server, `/gamemodes` to join

Gamemodes is a Rift-like dimension, hub for [minigames](#minigames)

### Archives

> **Note: Archives are rarely hosted.**

Reviewer: ```suggestion
**Note: Currently the Archives are not hosted 24/7.**
```

TSL Servers, `/archive0x5`, `/archive0x5b`, `/archive0x2`, `/archive0x3` or `/archive0x4` to join any of the 5 dimensions

Archives are comprised of 5 dimensions `archive0x5`, `archive0x5b`, `archive0x2`, `archive0x3` and `archive0x4`, they are pieces of the Dark Gaming History, they contain the old lobbies of Dark Gaming.

## Closed

### AutoSurvival

TSL Lobby, TShock Worlds, `/autosurvival` to join.

> AutoSurvival is a type of survival where the server hosts a survival for you, you can then download the world later on.

Reviewer: ```suggestion
AutoSurvival is a type of survival where the server hosts a survival for you, you can then download the world later on. AutoSurvival only existed to make use of spare server capacity, it may come back in the future if such an event arises.
```

## TSL

> Terraria Server Lite or TSL servers are similar to TShock in some ways, TSL was created by popstarfreas (Rofle) to minimize resource consumption. It is currently only available on DG.

Reviewer: ```suggestion
Terraria Server Lite or TSL servers are similar to TShock in some ways, TSL was created by popstarfreas (Rofle) to minimize resource consumption. It is completely separate from Terraria's source code and makes no use of it (more info available at [the knowledgebase](https://github.com/popstarfreas/TerrariaServerLite-KnowledgeBase). It is currently only available on DG.
```

> Terraria Server Lite or TSL servers are similar to TShock in some ways, TSL was created by popstarfreas(Rofle) to minimize resource consumption by only rendering a selected area and ignoring the rest. Its features include grouping people into 10. It is currently only available on DG.

Reviewer: ```suggestion
Terraria Server Lite or TSL servers are similar to TShock in some ways, TSL was created by popstarfreas (Rofle) to minimize resource consumption. It is currently only available on DG.
```

## TShock

> TShock is the generic Server API for Terraria it was developed y The Nyx Team. Common among popular servers like: Terraria.One, Aurora, Teeria, and Pedguin.

Reviewer: ```suggestion
TShock is the generic Server API for Terraria it was developed by The Nyx Team. Common among popular servers like: Terraria.One, Aurora, Teeria, and Pedguin.
```
